appservice/src/connectToGitHub.ts

```diff
@@ -108,13 +90,12 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient):
     }
 }
 
-async function showGitHubAuthPrompt(): Promise<void> {
-    const learnMore: string = localize('learnMore', 'Learn More');
-    const setupGithub: string = localize('setupGithub', 'You must give Azure access to your GitHub account.');
-    const input: string | undefined = await vscode.window.showErrorMessage(setupGithub, learnMore);
-    if (input === learnMore) {
-        // tslint:disable-next-line:no-unsafe-any
-        opn('https://aka.ms/B7g6sw');
+async function showGitHubAuthPrompt(node: AzureTreeItem): Promise<void> {
+    const invalidToken: string = localize('tokenExpired', 'Azure\'s GitHub token in invalid.  Authorize in the "Deployment Center"');
+    const goToPortal: string = localize('goToPortal', 'Go to Portal');
+    const input: string | undefined = await vscode.window.showErrorMessage(invalidToken, goToPortal);
+    if (input === goToPortal) {
+        node.openInPortal(`${node.id}/vstscd`);
```

Comment:
I would prefer if you use `client.id` rather than `node.id` - just in case someone passes in the wrong node.